fv3core/stencils/updatedzd.py

```diff
@@ -352,7 +368,7 @@ def __call__(
             self._wk,
         )
         self._apply_height_fluxes(
-            self.grid.area,
+            self._area,
```

Comment:
Because this class only uses a single member from the GridData class. Storing individual variables is better in that it tells you what the class actually uses, but is worse in that it increases the number of attributes you need to store. Because we're only using one attribute, that drawback goes away.